Wave1_data/1.preprocess_data/scripts/1.define_platemap.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook extracts a platemap of the data
+
+# In[1]:
+
+
+import pathlib
+
+import pandas as pd
+
+# In[2]:
+
+
+# absolute path to the raw data directory only works on this machine
```

Comment:
Recommend changing this comment since it looks like it doesn't apply here (these paths below are relative to the current directory).

Reply:
changed!